addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -107,19 +107,32 @@ const RatingPopupComposer = publicWidget.Widget.extend({
         const data = event.data;
 
         // Refresh the internal state of the widget
-        this.rating_avg = data.rating_avg;
-        this.rating_count = data.rating_count;
-        this.rating_value = data.rating_value;
+        this.rating_avg = data.rating_avg || data["mail.thread"][0].rating_avg;
```

Comment:
Can we still receive data the old way too here?

Reply:
Yes, some other controllers like `mail_update_message` and `portal_message_fetch` still send the old format.